build.gradle

```
@@ -12,12 +12,13 @@ description = 'Allows multiple SCM plugins to be used in a job.'
 
 group = 'org.jenkins-ci.plugins'
 
+// Follow https://github.com/jenkinsci/gradle-jpi-plugin#configuration
 jenkinsPlugin {
     // version of Jenkins core this plugin depends on, must be 1.420 or later
-    coreVersion = '1.580.1'
+    jenkinsVersion = '2.264'
     shortName = 'multiple-scms'
     displayName = 'Jenkins Multiple SCMs plugin'
-    url = 'https://wiki.jenkins-ci.org/display/JENKINS/Multiple+SCMs+Plugin'
+    url = 'https://plugins.jenkins.io/multiple-scms/'
```

this should be the github repo, it's used for documentation
```suggestion
    url = 'https://github.com/jenkins/multiple-scms-plugin'
```

---

the wiki content should be exported and added to the repo first before changing this URL https://jenkins-wiki-exporter.jenkins.io/

---

Thanks, I'll address the docs in another PR then :)

---

#27